platform-coop-toolkit/pinecone
# feat: updated breadcrumb styles and link styling

## Description

This PR updates standalone links and breadcrumbs to match the updated styles for link states.

## Steps to test

1. Go to the Netlify url for this Pull Request.
2. Check:
   * Link
   * Link List
   * Footer
   * Layouts / Home
   * Layouts / Page
   * Layouts / Term List

**Expected behavior:**

- Focus style should be a single border without a background change
- Active style should be a border with a background change
- Hover should be a underline change

## Related issues

#366 - This PR addresses Link List, but not Accordion.
#243 - Updates standalone links (like headers)
